spec/controllers/flood_risk_engine/enrollments/steps_controller/partnership_details_spec.rb

```diff
@@ -0,0 +1,64 @@
+require "rails_helper"
+module FloodRiskEngine
+  module Enrollments
+
+    describe StepsController do
```

Comment:
well experience is showing that making this assumption is dangerous. plus this is not a normal flow is it, got its own controller setting steps etc hasn't it ?